### PR TITLE
Fix partial update of the "inputAttr" option (TextEditor and FileUploader) [T938361]

### PR DESCRIPTION
**Pull request by DokaRus**
```js
  textEditor.option('inputAttr.title', 'new title');
  fileUploader.option('inputAttr.title', 'new title');
```

**Expected behavior**: new attribute applies to the input element
**Actual behavior**: an error occurs

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js [hunk 1]
```diff
@@ -131,3 +131,191 @@ module('the \'name\' option', () => {
...
+
+            assert.strictEqual($input1.attr('data-test'), 'changedValue', 'Attr was changed by API');
+            assert.strictEqual($input1.attr('data-anyattr'), 'anyvalue', 'New attr was set by API');
+        } finally {
+            $div1.remove();
+            $div2.remove();
+        }
+    });
+
+    test('name option should not conflict with inputAttr.name option', function(assert) {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js]: Shouldn't we cut this test?

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js [hunk 2]
```diff
@@ -131,3 +131,191 @@ module('the \'name\' option', () => {
...
+            assert.ok(typeof (instance2.option('inputAttr')) === 'object' && $.isEmptyObject(instance2.option('inputAttr')), 'Option is {} by default');
+            assert.strictEqual(instance1.option('inputAttr')['data-test'], 'test', 'Option sets to the widget on init');
+            assert.strictEqual($input1.attr('data-test'), 'test', 'Option sets to the widget input on init');
+
+            instance1.option('inputAttr', { 'data-test': 'changedValue', 'data-anyattr': 'anyvalue' });
+
+            assert.strictEqual($input1.attr('data-test'), 'changedValue', 'Attr was changed by API');
+            assert.strictEqual($input1.attr('data-anyattr'), 'anyvalue', 'New attr was set by API');
+        } finally {
+            $div1.remove();
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js]: We can not to remove it because fixture clears on every test start

Comment (San4es) [reply testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js]: But we append this element to `document.body`, not to `qunit-fixture`.

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js [hunk 3]
```diff
@@ -131,3 +131,191 @@ module('the \'name\' option', () => {
...
+        this.instance.option('value', '321');
+        assert.strictEqual(this.input.val(), '321');
+    });
+
+    test('the "inputAttr" option', function(assert) {
+        let $div1;
+        let $div2;
+
+        try {
+            $div1 = $('<div>', { id: 'testDiv1' }).appendTo($(document.body));
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js]: Can we use `qunit-fixture` here?

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js (hunk 2 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js]: I didn't refactor the transferred tests, thanks

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js (hunk 3 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js]: already done

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js (hunk 1 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js]: I've split it

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js (hunk 2 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js]: @San4es I've fixed it too

**Review by ksercs — APPROVED** [review]

**Review by vladkovl — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**